### DISCUSSION TITLE
Older versions that just work

### DISCUSSION THREAD

**Discussion by alan5ab** [mrlt8/docker-wyze-bridge#1243] [Q&A]
I have read all the issues people are having with the newer versions and wonder how many are still using older versions that still work for them. For me the sweet spot is camera version 4.61.0.3 (rtsp not enabled and everything else i could turn off) and bridge version 2.5.3 . I record 8 Wyze V3 cameras, 3 Hikvisions and a Reolink CX410 with Sentryshot NVR. The wyze cameras never drop out running these old versions, water getting into the lens is a different story.

**Comment by mrlt8** [comment]
v2.6.0 is considered very stable if you're only doing video. If you want the audio to stay in sync you'll need to update to v2.9.4+.

As for the firmware - anything released before 2024 was pretty solid. While the 2024 firmware still work, they seem to have several connection issues.
